src/routes/chains/chains.controller.spec.ts

```diff
@@ -193,4 +196,105 @@ describe('Chains Controller (Unit)', () => {
       expect(mockNetworkService.get.mock.calls[1][1]).toBe(undefined);
     });
   });
+
+  describe('GET /:chainId/about/master-copies', () => {
+    it('Success', async () => {
+      mockNetworkService.get.mockResolvedValueOnce({ data: chainResponse });
+      const domainMasterCopiesResponse: DomainMasterCopy[] = [
+        masterCopyFactory('test_address_1', 'test_version_1'),
+        masterCopyFactory('test_address_2', 'test_version_2'),
+      ];
+      mockNetworkService.get.mockResolvedValueOnce({
+        data: domainMasterCopiesResponse,
+      });
+      const masterCopiesResponse = [
+        <MasterCopy>{
+          address: 'test_address_1',
+          version: 'test_version_1',
+        },
+        <MasterCopy>{
+          address: 'test_address_2',
+          version: 'test_version_2',
+        },
+      ];
+
+      await request(app.getHttpServer())
+        .get('/chains/1/about/master-copies')
+        .expect(200)
+        .expect(masterCopiesResponse);
+
+      expect(mockNetworkService.get).toBeCalledTimes(2);
+      expect(mockNetworkService.get.mock.calls[0][0]).toBe(
+        'https://test.safe.config/api/v1/chains/1',
+      );
+      expect(mockNetworkService.get.mock.calls[1][0]).toBe(
+        `${chainResponse.transactionService}/api/v1/about/master-copies/`,
+      );
+      expect(mockNetworkService.get.mock.calls[1][1]).toBe(undefined);
+    });
+
+    it('Failure getting the chain', async () => {
+      mockNetworkService.get.mockRejectedValueOnce({
+        status: 400,
+      });
+
+      await request(app.getHttpServer())
+        .get('/chains/1/about/master-copies')
+        .expect(503)
+        .expect({
+          message: 'Service unavailable',
+          code: 503,
+        });
+
+      expect(mockNetworkService.get).toBeCalledTimes(1);
+      expect(mockNetworkService.get).toBeCalledWith(
+        'https://test.safe.config/api/v1/chains/1',
+        undefined,
+      );
+    });
+
+    it('Should fail getting the master-copies data', async () => {
+      mockNetworkService.get.mockResolvedValueOnce({ data: chainResponse });
+      mockNetworkService.get.mockRejectedValueOnce({
+        status: 502,
+      });
+
+      await request(app.getHttpServer())
+        .get('/chains/1/about/master-copies')
+        .expect(503)
+        .expect({
+          message: 'Service unavailable',
+          code: 503,
+        });
+
+      expect(mockNetworkService.get).toBeCalledTimes(2);
+      expect(mockNetworkService.get.mock.calls[0][0]).toBe(
+        'https://test.safe.config/api/v1/chains/1',
+      );
+      expect(mockNetworkService.get.mock.calls[1][0]).toBe(
+        `${chainResponse.transactionService}/api/v1/about/master-copies/`,
+      );
+      expect(mockNetworkService.get.mock.calls[1][1]).toBe(undefined);
+    });
+
+    it('Should return invalidate error', async () => {
```

Comment:
Do you mean "validation error" instead of "invalidate error" or am I getting it wrong? 🙂